### PR TITLE
Tweak remaining functions and update manuals

### PR DESCRIPTION
**Pull request by danielagarcia319**

### CODE REVIEW

File: R/generate_cre_dataset.R [hunk 1]
```diff
@@ -43,25 +43,26 @@ generate_cre_dataset <- function(n, rho, n_rules, effect_size, binary, seed) {
...
+    y0 <- rep(0, n)
+    y0[x2==1 & x3==1] <- 1
+    y1 <- rep(0, n)
+    y1[x1==0 & x2==0] <- 1
+    tau <- y1 - y0
   } else {
+    stopifnot(n_rules %in% c(2, 4))
+    if (n_rules == 2) {
+      tau = rep(0, n)
+      tau[x1==0 & x2==0] = effect_size
```
Comment (Naeemkh) [inline R/generate_cre_dataset.R:57]: Please use <- for assignments.

File: R/generate_rules.R [hunk 2]
```diff
@@ -20,14 +20,14 @@ generate_rules <- function(X, ite_std, ntrees, min_nodes, max_nodes) {
...
-                                       nodesize = min_nodes)
+  forest <- suppressWarnings(randomForest::randomForest(x = X, y = ite_std, sampsize = sf * N,
+                                                        replace = FALSE, ntree = 1, maxnodes = mn,
+                                                        nodesize = min_nodes))
   for(i in 2:ntrees) {
     mn <- 2 + floor(stats::rexp(1, 1 / (max_nodes - 2)))
-    model1_RF <- randomForest::randomForest(x = X, y = ite_std, sampsize = sf * N ,
-                                            replace = FALSE, ntree = 1, maxnodes = mn,
-                                            nodesize = min_nodes)
+    model1_RF <- suppressWarnings(randomForest::randomForest(x = X, y = ite_std, sampsize = sf * N ,
```
Comment (Naeemkh) [inline R/generate_rules.R:28]: Please keep codes in the 80 char limit.

File: tests/testthat/test-generate_cre_dataset.R [hunk 3]
```diff
@@ -1,19 +1,19 @@
 test_that("Synthetic Data Generated Correctly", {
   # Incorrect n input
-  expect_error(generate_cre_dataset(n = "test", rho = 0, n_rules = 2, effect_size = 0.5, binary = FALSE, seed = 2021))
+  suppressWarnings(expect_error(generate_cre_dataset(n = "test", rho = 0, n_rules = 2, effect_size = 0.5, binary = FALSE, seed = 2021)))
```
Comment (Naeemkh) [inline tests/testthat/test-generate_cre_dataset.R:3]: Please keep the codes in the 80 char limitation.

File: tests/testthat/test-generate_cre_dataset.R [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 
   # Incorrect rho input
-  expect_error(generate_cre_dataset(n = 1000, rho = "test", n_rules = 2, effect_size = 0.5, binary = FALSE, seed = 2021))
+  suppressWarnings(expect_error(generate_cre_dataset(n = 1000, rho = "test", n_rules = 2, effect_size = 0.5, binary = FALSE, seed = 2021)))
```
Comment (Naeemkh) [inline tests/testthat/test-generate_cre_dataset.R:6]: Why are you suppressing Warnings?

File: tests/testthat/test-cre.R [hunk 5]
```diff
@@ -1,46 +1,52 @@
 test_that("CRE Runs Correctly", {
   # Generate sample data
-  dataset_cont <- generate_cre_dataset(n = 1000, rho = 0, n_rules = 2, effect_size = 0.5, discrete = FALSE, seed = 2021)
+  dataset_cont <- generate_cre_dataset(n = 1000, rho = 0, n_rules = 2, effect_size = 0.5, binary = TRUE, seed = 2021)
```
Comment (Naeemkh) [inline tests/testthat/test-cre.R:3]: 80 char limitation!

File: tests/testthat/test-estimate_ps.R [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  # Incorrect data inputs
+  suppressWarnings(expect_error(estimate_ps(z = "test", X)))
+  suppressWarnings(expect_error(estimate_ps(z, X = "test")))
+
+  # Correct outputs
+  est_ps <- estimate_ps(z, X)
+  expect_true(length(est_ps) == length(z))
+  expect_true(class(est_ps) == "numeric")
+})
+
```
Comment (Naeemkh) [inline tests/testthat/test-estimate_ps.R:16]: Extra space.

File: tests/testthat/test-generate_cre_dataset.R (hunk 4 above)
Comment (danielagarcia319) [reply tests/testthat/test-generate_cre_dataset.R:6]: Just to simplify the output of testthat, but I can remove them!

**Review by Naeemkh — APPROVED** [review]

**Pull request merged**